package.json

```diff
@@ -25,5 +25,15 @@
   "bugs": {
     "url": "https://github.com/nelsonic/hapi-login/issues"
   },
-  "homepage": "https://github.com/nelsonic/hapi-login"
+  "homepage": "https://github.com/nelsonic/hapi-login",
+  "devDependencies": {
+    "bell": "^3.0.0",
+    "hapi": "^8.5.3"
+  },
+  "dependencies": {
+    "bell": "^5.1.0",
+    "hapi": "^10.0.0",
```

Comment:
@jalcine exactly. there has not been any change to the Plugin system since `v.8.x.x` see: https://github.com/dwyl/hapi-auth-jwt2#compatibility